models/olids/base/base_olids_diagnostic_order.sql

```
@@ -29,31 +35,30 @@ SELECT
-    src.lds_lakehouse_datetime_updated
-FROM {{ source('olids_common', 'DIAGNOSTIC_ORDER') }} src
+    src.publisher_organisation_code,
+    src.source_extraction_date,
+    src.lds_transform_datetime
+FROM {{ ref('landing_diagnostic_order') }} src
 INNER JOIN {{ ref('base_olids_patient') }} patients
     ON src.patient_id = patients.id
 INNER JOIN {{ ref('int_wnl_practices') }} wnl_practices
     ON src.publisher_organisation_code = wnl_practices.practice_code
-WHERE src.lds_start_datetime IS NOT NULL
+LEFT JOIN {{ ref('int_enriched_concept_map') }} concept_map
```

SQLFluff (4.2.2)

[error] Implicit/explicit aliasing of table.

(AL01)